specifyweb/frontend/js_src/lib/components/Formatters/Element.tsx

```diff
@@ -75,7 +81,9 @@ export function XmlEditorShell<
             {commonText.delete()}
           </Button.Danger>
           <span className="-ml-2 flex-1" />
-          <Submit.Info form={id('form')}>{commonText.close()}</Submit.Info>
+          <Submit.Info disabled={hasDuplicates} form={id('form')}>
```

Comment:
please remove `disabled={hasDuplicates}`
now that you set validation for Input.Text, making sure the submit button is NOT disabled is important.
Because, if some field on the form is invalid, then clicking the submit button will focus the invalid field - but not if the submit button is disabled